Admin/AbstractAdmin.php

```diff
@@ -422,8 +422,6 @@
      */
     protected $securityInformation = array();
 
-    protected $cacheIsGranted = array();
```

Comment:
@Soullivaneuh yes, and a very fresh one at that : https://github.com/sonata-project/SonataAdminBundle/pull/4284/files#diff-f06e49acf2d25569dcbac961750bdb33R113 (I like this one better, more mutualization).